interchain.go

```diff
@@ -226,7 +230,11 @@ func (ic *Interchain) Build(ctx context.Context, rep *testreporter.RelayerExecRe
 			)
 		}
 
-		if err := rp.Relayer.LinkPath(ctx, rep, rp.Path); err != nil {
+		// If channel creation options are not specified, default to the ics20 fungible token transfer channel options.
+		if (opts.CreateChannelOpts == ibc.CreateChannelOptions{}) {
+			opts.CreateChannelOpts = ibc.DefaultChannelOpts()
+		}
```

Comment:
What happens if the user only sets partial info? 

E.g. `CreateChannelOptions{Version: "ics123"}` and they don't set other fields like `Order`.

Reply:
What I'm hinting at is perhaps we want to default all fields to a reasonable value even if the user only sets some of the fields.

Reply:
Good point! I'll make those changes 👍